### DISCUSSION TITLE
动态修改配置是否还有比 configWillLoad 更早的时机

### DISCUSSION THREAD

**Discussion by loveshens** [eggjs/egg#5072] [Q&A] [answered]
定时器读取config配置时候，已经读取完毕，在configWillLoad修改配置时候已经错过了最佳时机

**Comment by atian25** [comment, marked as answer]
一个 workaround， https://github.com/atian25/egg-remote-config 仅供参考。

你如果是 schedule 场景，最好是自行写一个 schedule 的 策略，具体看 egg-schedule README

**Reply by loveshens** [reply]
非常感谢天天
